server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTableProcessor.java

```diff
@@ -182,7 +186,8 @@ public RecipientRewriteTableProcessor(RecipientRewriteTable virtualTableStore, D
     }
 
     public RecipientRewriteTableProcessor(RecipientRewriteTable virtualTableStore, DomainList domainList, MailetContext mailetContext) {
-        this(virtualTableStore, domainList, mailetContext, new ProcessingState(Mail.ERROR), !REWRITE_SENDER_UPON_FORWARD);
+        this(virtualTableStore, domainList, mailetContext, new ProcessingState(Mail.ERROR), !REWRITE_SENDER_UPON_FORWARD,
+            false);
```

Comment:
```suggestion
            !FORWARD_AUTOMATED_EMAILS);
```

That or delete `FORWARD_AUTOMATED_EMAILS` as it does not seem used if not for this

Reply:
Ok with the comment.

Please push your fixup ;-)